### DISCUSSION TITLE
bridged ap mode - assignment of MAC or IP to br0

### DISCUSSION THREAD

**Discussion by mjschrod** [RaspAP/raspap-webgui#728] [Q&A]
Hi!
first of all, I'd like to say thank you for this great software!

To be honest, I think this is rather a feature  request or a unknown configuration modification than a bug:
I want to expand my current network by using the "bridged AP mode" setting and letting my router doing the DHCP work.
For that I have dynamic dhcp ip address assignment based on the mac address of the clients.

Without raspap or raspap with dhcp mode this works as expected. The eth0 of the raspberry gets his IP, which is configured in the router.
By switching on AP in bridged mode, things are different. A new device br0 is defined by raspap. And this devices asks for the IP from the DHCP. It gets on, but of course it is another one than the IP of eth0.

Is it possible to assign to br0 the same MAC as eth0?
Or is it possible to assign to br0 a distinct MAC and where and how is it done?

The target is to make sure that the ethernet IP is a certain one, which is defined in the DHCP.

**Comment by billz** [comment]
According to the manpage of `systemd.netdev` a MAC address can be specified in the [NetDev] section. 
Just add your eth0 MAC to `/etc/systemd/network/raspap-bridge-br0.netdev`:

```
[NetDev]
Name=br0
Kind=bridge
MACAddress=
```
